api/jobs/models/step.go

```diff
@@ -1,5 +1,27 @@
 package models
 
+// VulnerabilityScan holds information about vulnerabilities found during scan
+// swagger:model VulnerabilityScan
+type VulnerabilityScan struct {
+	// Status of the vulnerability scan
+	//
+	// required: true
+	// Enum: Success,Missing
+	// example: Success
+	Status string `json:"status,omitempty"`
```

Comment:
I guess for `required` property - attr `omitempty` is not logical?

Reply:
Agree. Good finding!